### ISSUE TITLE
Minor correction in documentation about Network Equations in https://nrel-siip.github.io/PowerSimulationsDynamics.jl/stable/component_models/network/

### ISSUE THREAD

**Issue by amar-iastate** [NREL-SIIP/PowerSimulationsDynamics.jl#49]
**Is your feature request related to a problem? Please describe.**
This is related to the equations in documentation at https://nrel-siip.github.io/PowerSimulationsDynamics.jl/stable/component_models/network/
In power system literature, the complex quantities are represented by Capital alphabets. The equations (1), (6), (7) & (8) need to be updated. Also equation (1) should be `0 = i(x,v) - Y*v` and not `0 = i(x,v) - Y*x`.

**Comment by kdheepak** [comment]
Are the quantities represented here instantaneous?

**Comment by rodrigomha** [comment]
> Are the quantities represented here instantaneous?

No, they are phasors.

@amar-iastate Although I agree with the notation being Capital, I was looking here to differentiate matrices with vectors using that type of notation here. I will discuss a good strategy to avoid confusion here.

**Comment by amar-iastate** [comment]
No.. The quantities are meant to be the phasor representation. The way it is written looks like it is the instantaneous version, which is not the case.

**Comment by amar-iastate** [comment]
> > Are the quantities represented here instantaneous?
> 
> No, they are phasors.
> 
> @amar-iastate Although I agree with the notation being Capital, I was looking here to differentiate matrices with vectors using that type of notation here. I will discuss a good strategy to avoid confusion here.

I think matrices are usually bold capitals and vectors are bold small alphabets. Equations (2) - (5) seem to be written in the normal convention.
